src/bo4e_generator/schema.py

```diff
@@ -75,4 +75,7 @@ def get_version(namespace: dict[str, SchemaMetadata]) -> str:
     Get the version of the bo4e schemas.
     """
     # The chosen class is arbitrary. All bo's and com's should contain the same version information.
-    return namespace["Angebot"].schema_parsed["properties"]["_version"]["default"]
+    try:
+        return namespace["Angebot"].schema_parsed["properties"]["_version"]["default"]
+    except KeyError:
+        return "unknown"
```

Comment:
Doofe Frage, aber wir sind ja aktuell deutsch unterwegs im BO4E Standard. Macht es dann Sinn den Default-Wert auch deutsch zu setzen?

Reply:
Also bisher waren z.B. alle doc strings, die technischer Natur waren, auf Englisch. Und ich persönlich halte gerne alles in Englisch, was geht. Hierbei geht es nur um einen Version-string, siehe hier: https://github.com/Hochfrequenz/intermediate-bo4e-migration-models/blob/22acb2fef03cf808b75a5e4ea2ea0673418b3f75/src/ibims/bo4e/__version__.py